src/Query/Aggregations/Metric/Max.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AviationCode\Elasticsearch\Query\Aggregations\Metric;
+
+use AviationCode\Elasticsearch\Model\Aggregations\Metric\Max as MaxModel;
+
+class Max extends Metric
+{
+    /**
+     * @var string
+     */
+    private $field;
+
+    public function __construct(string $field)
```

Comment:
The Max aggregation also accepts an optional `missing` argument used when no value is found in elastic. 

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-max-aggregation.html#_missing_value_4

Reply:
@Douglasdc3 The reason I did NOT add it is that we didn't do it in the **Min** aggregation although it does have the same option. Should I go back and include it in the **Min** aggregation as well ?

Moreover, looking into reference doc(s) for the `Cardinality` aggregation which we implemented earlier, looks like it has a `precision_threshold` optional option that we can as well, but still forgot to add it on. Should I go back and include this one as well ? 

[Min Aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-min-aggregation.html)

[Cardinality Aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-cardinality-aggregation.html)